plugins/systemadmin/systemadmin.plugin.zsh

```diff
@@ -140,12 +140,16 @@ d0() {
 
 # gather external ip address
 geteip() {
-    curl http://ifconfig.me
+    echo "$(curl -s -S http://ipecho.net/plain)"
```

Comment:
A better alternative could be `curl https://icanhazip.com`, which uses SSL and the output contains the necessary newline character. 

Also, what's with the `-sS` flags? Silent + show-errors; isn't it the same with no flags?